sycl/source/detail/scheduler/commands.cpp

```diff
@@ -821,7 +821,7 @@ cl_int ExecCGCommand::enqueueImp() {
     RT::PiKernel Kernel = nullptr;
 
     if (nullptr != ExecKernel->MSyclKernel) {
-      assert(ExecKernel->MSyclKernel->get_context() == Context);
+      assert(ExecKernel->MSyclKernel->get_info<info::kernel::context>() == Context);
```

Comment:
```suggestion
      assert(ExecKernel->get_context() == Context);
```

Reply:
`ExecKernel` turned to be a `CGExecKernel` and not `cl::sycl::kernel`, so this change breaks the build as `CGExecKernel` has no `get_context` method.
Please, revert your last comment whit this change.